### DISCUSSION TITLE
Tailwindscore?

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [bootscore/bootscore#510] [General]
Hello Basti! 

Im a trully big fan of your theme, its like 2 years that im building my site with it and still wip xD 

But! Now im pretty fluid in terms of bootstrap classes, and can manage it pretty well, just consider its my first framework and im jumped into it without css knowledge at all. Your theme has opened eyes on general wordpress development and understanding. I figure out how many things are involved and how vast wordpress can be (quircky also).

My is a general question since i like how your theme is structured, and now im trying to land functions (like badges, etc) into tailwind flavoured theme (using tailpress). 

Why dont you jump on a hype train? I think personally bootstrap has ended it's lifecycle (you can do a lot of things) and my is a personal opinion ofcourse, but its really hard to obrain something really different looking with bootstrap. I know its not a youtube video discussion but im pretty sure with your experience you can tailor a pretty cool theme with it. There is tailpress or TW_ theme but lacking of an ecosystem that you made around bootscore. 

Idk im just suggesting you (with no experience ofcourse) to create a Tailwind Alpine.js Theme that works similarly to bootscore and extend your existing plugins. Im personally will pay for something like that (reasonable price ofcourse). So maybe.. its pretty good commercial idea for you also.

**Comment by justinkruit** [comment]
One of the hardest things with Tailwind is that it does require nodejs. Unfortunately most basic webhostings don't allow this, while with bootScore we have a PHP SCSS compiler doing most of the work.

Another big downside is how we are styling woocommerce. We do a lot with SCSS, extending the styles over woocommerce classes to not need to edit the templates as much making woocommerce updates easier. Something like this is possible with Tailwind, but way more work and requires people to still edit the css files etc instead of using the classes like Tailwind is supposed to.

In terms of styling Bootstrap, I personally mainly use the grid and general elements, and write a lot of styling myself already. I've tried Tailwind, but find the amount of classes needed for styling a bit of clutter in the html, and find it easier to let this be done by just writing css myself. But again, that's my own opinion.

**Comment by crftwrk** [comment]
"Many fancy UI libraries come and go, but Bootstrap is for life"

**Comment by ritchiemarkov** [comment]
shiiet  :D


My phone has collapsed my keyboard, don't mean to be so exhaustive.

Yes. So thank you a lot for clarifying. I didn't even considered woocommerce.but yeah now I understand how many things are depending on third parties.

Thanks a lot for clarification!

As always on point!

You guys lead me into dev actually.

So big big big thanks. <3
